e2e/config/vhd.go

```diff
@@ -94,6 +97,11 @@ func findLatestImageWithTag(imageID, tagName, tagValue string) (VHDResourceID, e
 		}
 		versions := page.Value
 		for _, version := range versions {
+			// skip images tagged with the no-selection tag, indicating they
+			// shouldn't be selected dynmically for running abe2e scenarios
+			if _, ok := version.Tags[noSelectionTagName]; ok {
```

Comment:
how/when are those tags added?

Reply:
manually, mainly for adhoc cases like this